pype/lib/lib_old.py

```diff
@@ -197,38 +197,6 @@ def add_tool_to_environment(tools):
     os.environ.update(env)
 
 
-@contextlib.contextmanager
-def modified_environ(*remove, **update):
```

Comment:
`modified_environ` is removed here, but `pype/plugins/launcher/Aport.py` is still using it. If Aport.py is obsolete, it should be removed (and its mentions from docs too).

Reply:
Very true. We can remove it completely, it is completely obsolete now.